package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "luna",
-  "version": "1.7.0-beta",
+  "version": "1.7.1-beta",
```

Comment:
Not a fix/patch but a minor feature so minor version should be bumped not patch.

Please bump to `1.8.0`